src/Controller/Back/UserController.php

```diff
@@ -127,6 +130,34 @@ public function exportInactiveAccounts(
         ]);
     }
 
+    #[Route('/desactiver', name: 'back_user_disable', methods: ['POST'])]
+    public function disableUser(
+        Request $request,
+        UserManager $userManager,
+    ): Response {
+        $data = $request->get('user_disable');
```

Comment:
cette méthode sera déprécié dans les futures versions de symfony, faudra être plus explicite
```
$request->request-> pour POST
$request->query-> pour GET
```
https://symfony.com/blog/new-in-symfony-7-4-request-class-improvements#deprecated-the-get-method